types/externs.d.ts

```diff
@@ -119,6 +119,8 @@ declare global {
       channel?: string
       /** Precomputed lantern estimates to use instead of observed analysis. */
       precomputedLanternData?: PrecomputedLanternData | null;
+      // Flag indicating the path to the budgets.json file for LightWallet
+      budgetsPath?: string | null;
```

Comment:
The pattern I personally like is making `budgets: Budgets.Json` on the `SharedFlagSettings` type and `budgetsPath` be just in the CLI type.

In the CLI we can load the `budgets` JSON for the user, but this way a node user can just directly pass in an object (and later, extension/devtools/LR users).